### PR TITLE
feat(indexes): add get_n_height_tips method to height index

### PR DESCRIPTION
**Pull request by jansegre**
### Motivation

PR #686 needs this method

### Acceptance Criteria

- Add new method to the height index that allows getting a list of blocks starting from the tip

### Checklist

- [x] If you are requesting a merge into `master`, confirm this code is production-ready and can be included in future releases as soon as it gets merged

### CODE REVIEW

File: hathor/indexes/height_index.py [hunk 1]
```diff
@@ -34,6 +34,12 @@ class IndexEntry(NamedTuple):
     timestamp: int
 
 
+class HeightInfo(NamedTuple):
+    """Used by a few methods to represent a (height, hash) tuple."""
+    height: int
+    hash: bytes
```
Comment (msbrogli) [inline hathor/indexes/height_index.py:40]: Maybe use `id: VertexId`?

File: hathor/indexes/memory_height_index.py [hunk 2]
```diff
@@ -68,6 +68,19 @@ def get(self, height: int) -> Optional[bytes]:
...
+        if n_blocks < 1:
+            raise ValueError('n_blocks must be a positive, non-zero, integer')
+        # highest height that is included, will be the first element
+        h_high = len(self._index) - 1
+        # lowest height that is not included, None if it reaches the genesis
+        _h_low = h_high - n_blocks
+        h_low: Optional[int] = None if _h_low < 0 else _h_low
+        # zip the height using a range that starts at h_high and goes down to 0 and a slice of the index with len of at
+        # most n_blocks, both the range and the slice are already on the order that will be returned
+        return [HeightInfo(h, i.hash) for h, i in zip(range(h_high, -1, -1),
```
Comment (msbrogli) [inline hathor/indexes/memory_height_index.py:85]: Why not `[HeightInfo(h, self._index[h]) for h in range(h_high, h_low, -1)]`?

**Review by msbrogli — CHANGES_REQUESTED** [review]

**Review by alexruzenhack — DISMISSED** [review]

File: hathor/indexes/height_index.py (hunk 1 above)
Comment (jansegre) [reply hathor/indexes/height_index.py]: Done.

File: hathor/indexes/memory_height_index.py (hunk 2 above)
Comment (jansegre) [reply hathor/indexes/memory_height_index.py]: Done.

File: hathor/indexes/height_index.py (hunk 1 above)
Comment (msbrogli) [reply hathor/indexes/height_index.py]: Thanks!

File: hathor/indexes/memory_height_index.py (hunk 2 above)
Comment (msbrogli) [reply hathor/indexes/memory_height_index.py]: Thanks!

**Review by msbrogli — APPROVED** [review]

**Review by alexruzenhack — APPROVED** [review]

**Pull request merged**